arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -209,15 +209,15 @@
           "nextLinkName": "nextLink"
         },
         "x-ms-examples": {
-          "List of virtual network rules": {
+          "List virtual network rules": {
             "$ref": "../examples/VirtualNetworkRulesList.json"
           }
         }
       }
     }
   },
   "definitions": {
-    "VnetFirewallRuleProperties": {
+    "VirtualNetworkRuleProperties": {
       "description": "Properties of a virtual network rule.",
       "type": "object",
       "properties": {
```

Comment:
I think having vnet is that description is fine, unless you feel strongly that it needs to change.